scripts.js

```diff
@@ -0,0 +1,226 @@
+class Quiz {
+  constructor(properties) {
+    this.preloader = properties.preloader;
+    this.correctRequestAnswers = properties.correctRequestAnswers;
+    this.allAnswersRequest = properties.allAnswersRequest;
+    this.correctDefaultAnswers = properties.correctDefaultAnswers;
+    this.questionRequest = properties.questionRequest;
+    this.requestLink = properties.requestLink;
+    this.checkResultBind = this.checkResult.bind(this);
+    this.start = properties.start;
+    this.next = properties.next;
+    this.previous = properties.previous;
+    this.slides = properties.slides;
+    this.restart = properties.restart;
+    this.container = properties.container;
+    this.form = properties.form;
+    this.formName = properties.formName;
+    this.buttonName = properties.buttonName;
+    this.counterAnswers = properties.counterAnswers;
+    this.regex = properties.regex;
+    this.countSlide = 0;
+    this.countAnswers = 0;
+    this.timerShowQuestion;
+    this.timerCheckQuestion;
+  }
+
+  makeRequest() {
+    fetch(this.requestLink)
+      .then(res => res.json())
+      .then(json => {
+        let result = [...json.results];
+        let quizCategory = result[0].category;
+        result.forEach(obj => {
+          this.correctRequestAnswers.push(obj.correct_answer);
+          this.questionRequest.push(obj.question);
+          this.allAnswersRequest.push(...obj.incorrect_answers, obj.correct_answer);
+        });
+
+        document.querySelector('.title').textContent = `Quiz about ${quizCategory}`;
+
+        this.insertText(this.questionRequest, this.allAnswersRequest);
+        json.onload = this.preloader.classList.add('none');
+      })
+      .catch(err => console.log(err));
+  }
+
+  insertText(arrQuestion, arrAnswer) {
+    const questionText = this.container.querySelectorAll('.question__title');
+    for (let i = 0; i < arrQuestion.length; i++) {
+      questionText[i].textContent = arrQuestion[i];
+    }
+
+    const answerText = this.container.querySelectorAll('.answer');
+    const valueText = this.container.querySelectorAll('.radio');
+    for (let i = 0; i < arrAnswer.length; i++) {
+      answerText[i].textContent = arrAnswer[i];
+      valueText[i].value = arrAnswer[i];
+    }
+  }
+
+  assignHandlerAnswer() {
+    Array.from(this.container.querySelectorAll('.question .question__answers'))
+      .forEach(answers => {
+        answers.addEventListener("click", this.checkResultBind);
+      })
+  }
+
+  assignHandlerButton() {
+    this.start.addEventListener("click", this.startGame.bind(this));
+    this.next.addEventListener("click", this.switchNextQuestion.bind(this));
+    this.previous.addEventListener("click", this.switchPreviousQuestion.bind(this));
+    this.buttonName.addEventListener("click", this.eventButtonName.bind(this));
+    this.restart.addEventListener("click", this.eventRestart.bind(this));
+  }
+
+  checkResult(event) {
+    let target = event.target;
+    if (target.tagName != 'LABEL') return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705696)
```suggestion
    if (target.tagName != 'LABEL') {return;}
```